### DISCUSSION TITLE
memory stats in summary report missing in artillery-engine-playwright

### DISCUSSION THREAD

**Discussion by simplicity360** [artilleryio/artillery#1677] [Artillery]
Hi Artillery team, 
  I am writing some browser tests with artillery-engine-playwright.
According to README, the summary report should have shown **browser.memory_used_mb.**

However, the summary report I see has not such stat.

Should I emit a customStat, if so can someone please guide me here.

Thanks.

**Comment by AtefBellaaj** [comment]
I am facing the same issue. :(

**Reply by bernardobridge** [reply]
Hello @AtefBellaaj 👋 . 

Try setting the `extendedMetrics` setting to `true` in your config:
```yaml
config:
  engines:
    playwright:
      extendedMetrics: true
```

We'll fix the documentation to make this clear - apologies for the confusion.

**Reply by AtefBellaaj** [reply]
Hello @bernardobridge 
Thank you very much, this works.
